CHANGELOG.rst

```diff
@@ -1,5 +1,10 @@
 Changelog for Mantarray Frontend Components
 ===========================================
+0.1.3 (2020-12-14)
+------------------
+
+- Updated a Error Catch Handling
```

Comment:
please provide more details---how did this update change the behavior?

Reply:
Now its:-
`- Updated Error Handling capability via ErrorCatchWidget and gracefully shutdown for electron app`